docs/visual-testing/workflows/review.md

```diff
@@ -124,6 +124,24 @@ There is also an option to reject the snapshot. This will mark the snapshot as "
 
 More information about reviewing and approving diffs can be found in [Subsequent Test Execution Review](#subsequent-test-execution-review).
 
+##### Diff inspection
```

Comment:
Title should be in Title case, and also have a DOM keyword added:
DOM Diff Inspection